neutrino-custom/lint.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+const merge = require('deepmerge');
+const lintBase = require('neutrino-lint-base');
+const path = require('path');
+
+const CWD = process.cwd();
+const UI = path.join(CWD, 'ui');
+
+module.exports = neutrino => {
+    lintBase(neutrino);
+    neutrino.config.module
+        .rule('lint')
+        .include(UI)
+        .test(/\.jsx?$/)
+        .loader('eslint', props => merge(props, {
+            options: {
+                plugins: ['react'],
+                envs: ['browser', 'es6', 'node'],
+                parserOptions: {
+                    sourceType: 'script',
+                    ecmaFeatures: {
+                        es6: true,
+                        jsx: true,
+                        impliedStrict: false
+                    }
+                },
+                extends: 'eslint:recommended',
+                rules: {
+                    'accessor-pairs': 2,
```

Comment:
Something that would help the readability of these rules would be to replace all `2`s with `'error'`.